src/EditorFeatures/TestUtilities/Formatting/CoreFormatterTestsBase.cs

```diff
@@ -40,20 +36,8 @@ public abstract class CoreFormatterTestsBase
             var snapshot = subjectDocument.TextBuffer.CurrentSnapshot;
             var indentationLineFromBuffer = snapshot.GetLineFromPosition(point);
 
-            var commandHandler = CreateSmartTokenFormatterCommandHandler(textUndoHistory.Object, editorOperationsFactory.Object);
-            commandHandler.ExecuteCommandWorker(new ReturnKeyCommandArgs(textView, subjectDocument.TextBuffer), CancellationToken.None);
-            var newSnapshot = subjectDocument.TextBuffer.CurrentSnapshot;
-
-            int? actualIndentation;
-            if (newSnapshot.Version.VersionNumber > snapshot.Version.VersionNumber)
-            {
-                actualIndentation = newSnapshot.GetLineFromLineNumber(indentationLineFromBuffer.LineNumber).GetFirstNonWhitespaceOffset();
-            }
-            else
-            {
-                var provider = new SmartIndent(textView);
-                actualIndentation = provider.GetDesiredIndentation(indentationLineFromBuffer);
-            }
```

Comment:
so this is another behavior change. 

this is like so many behaviors changes you guys were like fighting over me to never change. I am not sure how this suddenly okay. this is too much even for me.

I am not sure why you suddenly decide to remove editor option support from C# and completely change our behavior on "enter"

Reply:
> I am not sure why you suddenly decide to remove editor option support from C# and completely change our behavior on "enter"

It doesn't completely change things.  There was one known place where C# had *very* special handling on enter.  Specifically only in a `using`-statement.  That special case was taken to a design meeting and it was decided:

1. we didn't need this very special enter-handling case.
2. the way the `using-statement` auto formatted through normal typing was appropriate and sufficient.